src/test/unit/controller/NoticeEndController.test.ts

```diff
@@ -17,4 +17,12 @@ describe('Notice end Controller', () => {
     noticeEndController.get(request, response);
     expect(response.render).toHaveBeenCalledWith(TranslationKeys.NOTICE_END, expect.anything());
   });
```

Comment:
Please include a test for a successful post

Reply:
Added

Reply:
Not seeing a post test added here